pom.xml

```diff
@@ -352,6 +360,83 @@
       </build>
     </profile>
 
+    <profile>
+      <!-- This profile is used to enable GraalVM native image testing
+      (.kokoro/graalvm-native/linux) -->
+      <id>native</id>
+
+      <dependencyManagement>
+        <!-- Controlling version for dependencyConvergence enforcer rule -->
+        <dependencies>
+          <dependency>
+            <groupId>org.junit.platform</groupId>
+            <artifactId>junit-platform-engine</artifactId>
+            <version>1.8.2</version>
+          </dependency>
+          <dependency>
+            <groupId>org.junit.platform</groupId>
+            <artifactId>junit-platform-commons</artifactId>
+            <version>1.8.2</version>
+          </dependency>
+        </dependencies>
+      </dependencyManagement>
```

Comment:
Now "native" profile has this dependencyManagement section to control the enforcer rule.

Reply:
Nice! Thanks for doing that.